fedimint-server/src/atomic_broadcast/backup.rs

```diff
@@ -20,7 +20,9 @@ pub async fn load_session(db: Database) -> (Cursor<Vec<u8>>, UnitSaver) {
 
     std::mem::drop(dbtx);
 
-    tracing::info!(target: LOG_CONSENSUS,"Loaded aleph buffer with {} bytes", buffer.len());
+    if !buffer.is_empty() {
+        tracing::info!(target: LOG_CONSENSUS,"Loaded aleph buffer with {} bytes (we are recovering from an in-session shutdown)", buffer.len());
```

Comment:
Would write `bytes = buffer.len(), "Recovering from an in-session-shutdown"`.

Reply:
changed